.github/workflows/docker-build.yml

```
@@ -344,14 +344,14 @@ jobs:
 
       - name: Login to Docker Hub
         if: ${{ matrix.component == 'docker-backend' }} || ${{ matrix.component == 'docker-frontend' }}
```

actionlint (1.7.11)

[error] if: condition "${{ matrix.component == 'docker-backend' }} || ${{ matrix.component == 'docker-frontend' }}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)